doc/api/process.md

```diff
@@ -2010,7 +2024,6 @@ cases:
 [`'exit'`]: #process_event_exit
 [`'finish'`]: stream.html#stream_event_finish
 [`'message'`]: child_process.html#child_process_event_message
-[`'rejectionHandled'`]: #process_event_rejectionhandled
```

Comment:
?

Reply:
It is now unused because I changed some docs above.

Reply:
Ok, the hook wasn't removed though right?

Reply:
I'm +1 on removing the recommendation to use the hook by the way. Just to clarify.

Reply:
No, the hook is still there.